assignments/objects.js

```diff
@@ -18,27 +18,66 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const mitzi = {
+  "id": 1,
+  "name":"Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak : function(){
+    return "Hello , my name is Kennan"
+  }
```

Comment:
Really great job on the speak function. You could also have written it separately.